src/GraphQL/Types/Fields/FieldType.cs

```diff
@@ -91,5 +91,12 @@ internal void SetName(string name, bool validate)
         /// Gets or sets a subscription resolver for the field. Only applicable to the root fields of subscription.
         /// </summary>
         public ISourceStreamResolver? StreamResolver { get; set; }
+
+        /// <summary>
+        /// Validates and/or parses the value received from the client.
+        /// Throw an exception if necessary to indicate a problem.
+        /// Only applicable to fields of input graph types.
+        /// </summary>
+        public Func<object?, object?> ParseValue { get; set; } = static value => value;
```

Comment:
Should this be an interface with a property, like `IFieldResolver` is?  Not sure exactly what the benefit would be.  At least it would match the `Resolver` and `StreamResolver` properties.